.travis.yml

```diff
@@ -46,33 +46,68 @@ env:
   - TEST_TARGET="validate"                           TW="travis_wait"
   - TEST_TARGET="validate"   COMPILER="4.02.3+32bit" TW="travis_wait"
   - TEST_TARGET="validate"   COMPILER="${COMPILER_BE}+flambda" CAMLP5_VER="${CAMLP5_VER_BE}" NATIVE_COMP="no" EXTRA_CONF="-flambda-opts -O3" EXTRA_OPAM="num" FINDLIB_VER="${FINDLIB_VER_BE}"
-  - TEST_TARGET="ci-bignums"
-  - TEST_TARGET="ci-color"
-  - TEST_TARGET="ci-compcert"
-  - TEST_TARGET="ci-coq-dpdgraph" EXTRA_OPAM="ocamlgraph"
-  - TEST_TARGET="ci-coquelicot"
-  - TEST_TARGET="ci-equations"
-  - TEST_TARGET="ci-geocoq"
-  - TEST_TARGET="ci-fiat-crypto"
-  - TEST_TARGET="ci-fiat-parsers"
-  - TEST_TARGET="ci-flocq"
-  - TEST_TARGET="ci-formal-topology"
-  - TEST_TARGET="ci-hott"
-  - TEST_TARGET="ci-iris-lambda-rust"
-  - TEST_TARGET="ci-ltac2"
-  - TEST_TARGET="ci-math-classes"
-  - TEST_TARGET="ci-math-comp"
-  - TEST_TARGET="ci-sf"
-  - TEST_TARGET="ci-unimath"
-  - TEST_TARGET="ci-vst"
-  # Not ready yet for 8.7
-  # - TEST_TARGET="ci-cpdt"
-  # - TEST_TARGET="ci-metacoq"
-  # - TEST_TARGET="ci-tlc"
 
 matrix:
 
   include:
+    - if: NOT (type = pull_request)
```

Comment:
Ok, let's ask the obvious: did you try to factorize the conditional? I thought the indentiation would express the scope.

Reply:
So the answer is yes, obviously I first tried to factorize but didn't manage. However, I did not dig into the yaml / Travis doc so I might have missed some way of doing it. Maybe @SkySkimmer can tell if he thinks that this would have been possible.

Reply:
>So the answer is yes, obviously I first tried to factorize but didn't manage. However, I did not dig into the yaml / Travis doc so I might have missed some way of doing it. Maybe @SkySkimmer can tell if he thinks that this would have been possible.

I don't think it's possible to factorize with jobs.
If you put all the conditionally executed jobs in a stage you can put a condition on the whole stage, and putting jobs in a stage can be factorized because you only need to specify the stage for the first job in the list.